verified_boot_app.md

```diff
@@ -11,8 +38,108 @@ Src: https://github.com/Kicksecure/kicksecure-wiki-backup/blob/6900fdfbdea8693fd
 * Generic concept: yes
 * Architecture specific implementation: Yes, initially <code>amd64</code> only.
 * Architecture support: Initially <code>amd64</code> only. In principle, can be ported to other architectures by third parties.
-* Must be a motherboard deemed commercially profitable by the OEM: yes.
-* Must be a feature that the firmware developer considers potentially commercially beneficial (as in customers would buy motherboards with such freedom-preserving firmware): yes
+
+-->
+
+1.1. The application MUST be compatible with UEFI 2.7 Specification. Any
+     system compliant with the aforementioned specification version SHOULD be
+     capable of running this application.
+
+1.2. The application SHALL NOT support systems running in legacy
+     BIOS/compatibility mode (CSM).
+
+1.3. The application SHOULD be tested and verified on AMD64 architecture only.
+     However, the architecture support MAY NOT be limited to AMD64 only and
+     MAY also work on any architecture supported by the UEFI Specification 2.7
+     (refer to Section 2.3 Calling Conventions).
+
+<!--
+* Vendor neutral: yes, any operating system that wishes to be compatible with the "EFI_ALT" standard/convention is welcome to.
+* Kicksecure specific: no
+* Kicksecure branding: no
+-->
+
+1.4. The application MUST be OS/HW vendor agnostic and NOT subject to any
+     branding.
```

Comment:
we have to decide if we prevent branding or we at this point allow branding, allowing branding can improve adoption IMHO

Reply:
Can we make this dual layer?

* 1) Keep the specification agonist.
* 2) The implementation itself might be branded: "`$vendor_specific_feature based on $specification`"?

There is no need to say anything about branding at this stage. Perhaps simply drop this sentence?

Reply:
I think we should take branding into account - if we don't, more likely than not the finished product will end up un-brandable which could be a dealbreaker for some vendors. The ability to at least include the vendor's name in a somewhat easy fashion (changing a configuration file or even a C `#define` at build time) would be nice and should be sufficient for a text-only interface. If the UI is going to be graphical, the ability to add a logo too would be nice, but I don't think this needs to be graphical.

Reply:
Yes, we are not discussing implementation yet, nor are mockups needed anyway, but after closing the spec stage.

I would say the application MAY be brandable, but I guess the ultimate decision should be to @adrelanos ?

Reply:
Answered in matrix a while ago.

MAY is ok. 3MDEB shall decide.